tensorboard/webapp/metrics/views/main_view/card_grid_component.ts

```diff
@@ -49,6 +55,16 @@ export class CardGridComponent {
     @Optional() private readonly cdkScrollable: CdkScrollable | null
   ) {}
 
+  ngOnInit() {
+    if (
+      this.cardMaxWidthInVW &&
+      this.cardMaxWidthInVW >= MIN_CARD_MAX_WIDTH_IN_VW &&
```

Comment:
Short answer is no.  The range of the slider should be 30 to 100 (unit vw). And we will not show the numbers to preserve interpretation (instead of explicitly explaining what the numbers are)